lib/setup.py

```diff
@@ -106,7 +106,7 @@ def run(self):
     },
     author="Streamlit Inc",
     author_email="hello@streamlit.io",
-    python_requires=">=3.7",
+    python_requires=">=3.7, !=3.9.7",
```

Comment:
Can we add a comment (maybe just link to the Github issue) that explains why this is excluded?

Reply:
Will do